### PR TITLE
For comparison until the next release

### PR DESCRIPTION
**Pull request by maelle**
@sckott will be in charge of the next release.

### CODE REVIEW

**Comment by sckott** [comment]
@maelle some questions about how to do the next release:

- i assume the next release is https://github.com/ropensci/dev_guide/milestone/2 ?
- do we add a news entry for all changes no matter how small, e.g, https://github.com/ropensci/dev_guide/pull/182 
- I forget, was there any info on how to do the release? i think we're supposed to do a blog post?

**Comment by maelle** [comment]
- correct milestone indeed
- not typos i think, but small things can be listed in the NEWS (but you can ignore them in the announcement blog post, pointing to the full changelog section in the book)
- yes see https://devdevguide.netlify.com/editorguide.html#bookrelease with the checklist in https://devdevguide.netlify.com/bookreleaseissue.html it is still in the dev version hence your not finding it easily. 😁

**Comment by sckott** [comment]
@maelle thanks!

**Comment by sckott** [comment]
@ropensci/editors ready for review - see also blog post https://github.com/ropensci/roweb2/pull/521

File: appendix.Rmd [hunk 1]
```diff
@@ -2,8 +2,52 @@
 
 # NEWS {#booknews}
 
+## dev
+
+* 2019-09-26, add instructions for handling editors to nominate packages for blog posts (ropensci/dev_guide#180)
+
+* 2019-09-26, add chapter on changing package maintainers (ropensci/dev_guide#128) (ropensci/dev_guide#194)
```
Comment (maelle) [inline appendix.Rmd]: Why put ropensci/dev_guide instead of just the PR number? Genuinely curious.

Comment (sckott) [reply appendix.Rmd]: we have some issues that are from software-review-meta and some from dev_guide. so unless there's a org/repo before the issue number, there's no way to tell which repo the issue is from

**Comment by sckott** [comment]
Ready to merge when I get a few approvals

**Pull request merged**